portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase

```diff
@@ -109,14 +109,35 @@ definition {
 	}
 
 	@description = "Verify that the Field Label can be edited"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test LabelCanBeEdited {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133039 LabelCanBeEdited pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Image",
+			formPageNumber = "0");
+
+		Form.editFieldLabelMetal(fieldValue = "Test Image");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
 
+		FormFields.viewRadioLabel(
+			fieldLabel = "Test Image",
+			fieldName = "Test Image");
```

Comment:
Hi @chaowang-123,

I think `FormFields.viewLabel` macro would fit better in this case, just by name consistency. Although `viewRadioLabel` macro has the same code of `viewLabel`, this "radio" it a first glance feels wrong, since we testing an Image field.

Thanks =)

Reply:
hi, @alinedoleron , macro has been changed, thanks~